docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,86 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
+
+### Opaque Integer Resources (Alpha Feature)
```

Comment:
One or the other. Either don't have an Alpha Features heading (and mark features Alpha), or don't include "Alpha" in each feature's individual heading.

Also, avoid nesting headings directly under one another; that's indicative of structural problems. If you have a header with nothing under it but another header, one or the other is not necessary.